hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/HawkbitUIErrorHandler.java

```diff
@@ -38,10 +42,21 @@ public void error(final ErrorEvent event) {
 
         final Optional<Page> originError = getPageOriginError(event);
 
+        final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
+
         if (originError.isPresent()) {
-            final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
+            final Connector connector = ((ConnectorErrorEvent) event).getConnector();
+            // in case of BulkUpload: BulkUpload needs a lock to show the
```

Comment:
Do not mention a specific view here. How do you know in that line that the view even exists?